src/core_modules/capture-core/components/WidgetsRelationship/WidgetTrackedEntityRelationship/NewTrackedEntityRelationship/hooks/useAddRelationship.ts

```diff
@@ -8,17 +7,11 @@ import { handleAPIResponse, REQUESTED_ENTITIES } from 'capture-core/utils/api';
 type Props = {
     teiId: string;
     onMutate?: () => void;
-    onSuccess?: (apiResponse: Object, requestData: Object) => void;
-}
+    onSuccess?: (apiResponse: Record<string, unknown>, requestData: Record<string, unknown>) => void;
+};
 
 const ReactQueryAppNamespace = 'capture';
 
-const addRelationshipMutation = {
-    resource: '/tracker?async=false&importStrategy=CREATE',
-    type: 'create',
-    data: ({ apiData }) => apiData,
-};
-
```

Comment:
Add back the deleted const addRelationshipMutation and import and use the type Mutation from 'capture-core-utils/types/app-runtime';
```
const addRelationshipMutation: Mutation = {
    resource: '/tracker?async=false&importStrategy=CREATE',
    type: 'create',
    data: ({ apiData }) => apiData,
};
```